environments/skeleton/{{cookiecutter.environment}}/terraform/control.userdata.tpl

```diff
@@ -0,0 +1,21 @@
+#cloud-config
+disk_setup:
```

Comment:
If there's just a single partition per-device, should we just create the filesystem directly on the device rather than partitioning first?